src/main/webapp/index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
Although not required, the Google HTML style guide recommends the omission of optional tags, such as `<html>`, `<head>`, and `<body>`.

https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags